pkg/utils/errors/error.go

```diff
@@ -0,0 +1,50 @@
+package errors
+
+import (
+	"fmt"
+)
```

Comment:
Single import shouldn't be declared as a block.

```suggestion
	import "fmt"
```